src/templates/nl/base.html

```diff
@@ -143,17 +147,17 @@
 {% block appendix %}Bijlage{% endblock %}
 {% block appendices %}Bijlagen{% endblock %}
 
-{% block ebook_title %}Ebook{% endblock %}
-{% block ebook_download_short %}Ebook PDF ({{ ebook_size_in_mb }}MB){% endblock %}
+{% block ebook_title %}E-book{% endblock %}
+{% block ebook_download_short %}E-book PDF ({{ ebook_size_in_mb }}MB){% endblock %}
 {% block ebook_download %}Download de volledige {{ year }} Web Almanac in PDF-formaat ({{ ebook_size_in_mb }}MB){% endblock %}
-{% block ebook_download_note %}(generated with <a href="https://www.princexml.com/">www.princexml.com</a>){% endblock %}
+{% block ebook_download_note %}(gegenereerd met <a href="https://www.princexml.com/">www.princexml.com</a>){% endblock %}
 
 {%
   set localizedPartTitles = {
-    "I. Page Content": "I. Pagina Inhoud",
+    "I. Page Content": "I. Paginainhoud",
```

Comment:
```suggestion
    "I. Page Content": "I. Inhoudsopgave",
```

Reply:
This is a translation for [this part name](https://almanac.httparchive.org/en/2021/table-of-contents#part-1), I don't think this means "table of contents"?

Reply:
True, then it'd be "Pagina inhoud"

Reply:
But it's a compound noun ("inhoud van een pagina", "content of a page"), so it should be written as one word, no?

Reply:
True but a & i can't be right next to eachother because of 'klinkerbotsing' so then it should be Pagina-inhoud

Reply:
Yep, you're right! Now it should be good :)